runtime/v2/bundle.go

```diff
@@ -115,16 +116,24 @@ type Bundle struct {
 }
 
 // Delete a bundle atomically
-func (b *Bundle) Delete() error {
+func (b *Bundle) Delete(ctx context.Context) error {
 	work, werr := os.Readlink(filepath.Join(b.Path, "work"))
 	rootfs := filepath.Join(b.Path, "rootfs")
+
+	// on windows hcsshim writes panic logs in the bundle directory in a file named
+	// "panic.log" log those messages (if any).
+	logs, err := ioutil.ReadFile(filepath.Join(b.Path, "panic.log"))
```

Comment:
Should we distinguish between not found errors and others here?

Reply:
I don't think there is any need for that. Because if there are any errors then we are not going to log anything. We log the contents only if there are no errors and the file is not empty. What do you think?

Reply:
If we don't think knowing that we failed to transfer the panic log is helpful or if we are confident it would never happen then best effort is enough.